src/Relation/Binary/Reasoning/PartialSetoid.agda

```diff
@@ -0,0 +1,24 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Convenient syntax for reasoning with a partial setoid
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Relation.Binary
+
+module Relation.Binary.Reasoning.PartialSetoid {s₁ s₂} {A : Set s₁} (S : PartialSetoid A s₂) where
+
+open PartialSetoid S
+
+------------------------------------------------------------------------
+-- Publicly re-export base contents
+
+open import Relation.Binary.Reasoning.Base.Partial _≈_ trans public
```

Comment:
You haven't actually included this file in the PR :smile:

Reply:
Just goes to show that I shouldn't be allowed to commit before I've had my coffee